src/mobster/cli.py

```diff
@@ -227,29 +227,73 @@ def augment_command_parser(subparsers: Any) -> None:
 
     """
     augment_parser = subparsers.add_parser(
-        "augment", help="Augment an SBOM document with additional information"
+        "augment",
+        help="Augment SBOM documents with additional information from a mapped"
+        "snapshot spec and save them to a directory.",
     )
+
     augment_subparser = augment_parser.add_subparsers(dest="type", required=True)
     augment_oci_image_parser(augment_subparser)
+    augment_snapshot_parser(augment_subparser)
 
 
 def augment_oci_image_parser(subparsers: Any) -> None:
     """
     A parser for augmenting SBOMs documents for OCI image.
     """
     oci_image_parser = subparsers.add_parser(
-        "oci-image", help="Augment an SBOM document for OCI image"
+        "oci-image", help="augment an SBOM document for an OCI image."
     )
     oci_image_parser.add_argument(
-        "--snapshot-path",
-        required=True,
+        "snapshot",
+        type=Path,
+        help="path to the mapped snapshot spec file in JSON format",
+    )
+    oci_image_parser.add_argument(
+        "reference",
+        type=str,
+        help="OCI image reference in the format <repository>@<digest>.",
+    )
+    oci_image_parser.add_argument(
```

Comment:
Can you move this argument one level above? The option is shared by all augment sub-parsers and we should align the CLI interface with the existing parser `generate_command_parser`